sips/sip-028/sip-028-sbtc_peg.md

```diff
@@ -0,0 +1,242 @@
+2. **Evaluation & Community Feedback**: The proposed signer set will be published to provide transparency.
+3. **SIP Vote**: The community will vote on the sBTC signer criteria.
+4. **Final Selection**:  If SIP-028 is ratified, then the proposed signer set voted upon in step 3 shall be the initial signing set for sBTC. If the signer set changes during the vote, such as by the withdrawal of one or more candidates, then the vote will be restarted in a subsequent reward cycle (to be determined if this comes to pass).
+
+   
+### Updating The sBTC Signer Set
+In the event that the sBTC Signer Set needs to be updated (for example, if a signer is no longer available to complete their responsibilities) sBTC Signers can perform a threshold vote to agree on the updated set. This process will also be performed if a signer needs to rotate their cryptographic keys.
+
+
+## Related Work
+
+### [WBTC](https://wbtc.network/assets/wrapped-tokens-whitepaper.pdf)
+
+**WBTC** is a closed membership system made up of 50+ merchants and custodians with keys to the WBTC multisig contract on Ethereum. WBTC deposits and withdrawals can only be performed by the authorized merchants, and end users purchase WBTC directly from the merchants. Until recently, BTC was held in a Bitcoin multi-sig wallet secured solely by BitGo. Now, two of the three multi-sig keys are held by BitGo, while one is held by BiT Global.
+
+### [tBTC v2](https://whitepaper.io/document/691/tbtc-whitepaper)
+
+**tBTC** is an ERC-20 wrapped asset launched in May 2020. BTC is currently held and secured by a permissioned set of 35 Beta Staker Nodes from the Threshold Network. Seven DeFi protocols including Aave and Synthetix manage the minting and burning process, with Guardians monitoring to veto suspicious behavior. tBTC is natively minted on Ethereum and Arbitrum.
+
+### [RBTC](https://rootstock.io/static/a79b27d4889409602174df4710102056/RS-whitepaper.pdf)
+
+**rBTC** is a wrapped BTC asset natively minted on Rootstock, an EVM-compatible sidechain. BTC is secured by a 5-of-9 multi-sig Bitcoin wallet controlled by the Powpeg Federation. Peg operations settle to Bitcoin via merge mining. Instead of collateralizing the system with a new token, peg operators are incentivized by earning a portion of transaction fees. PowPeg operators keep specialized hardware called PowHSMs active and connected to special types of Rootstock full nodes. Since the Bitcoin blockchain and the Rootstock sidechain are not entangled in a single blockchain or in a parent-child relation, peg-in and peg-out transactions require a high number of block confirmations. Peg-ins require 100 Bitcoin blocks, and peg-outs require 4000 Rootstock blocks (roughly 200 Bitcoin Blocks).
+
+This new system shares similarities with existing models but introduces some key distinctions:
+- **Decentralized Custody:** sBTC is secured by a decentralized network of signers rather than a central custodian.
```

Comment:
Of all three points, this one is the weakest.  `tBTC` and `rBTC` both have a distributed set of signers as well, and per this SIP, sBTC is controlled by a closed set of signers.  Is there something you could add here that further distinguishes sBTC from these two systems?

I wonder if a table would be more succinct.  `?` fields are information I don't know or can't easily find:

| **Feature** | **WBTC** | **tBTC** | **rBTC** | **sBTC** (this SIP) |
| --------------- | -------------- | ------------| ------------ | ------------------------- |
| Spending threshold | 2 of 3 | 51 of 100 | 5 of 9 | 11 of 15 |
| Bitcoin finality | no  |  no  |   yes |  yes |
| Expected Peg-in speed | ? | 3-4 hours | 16 hours | 0.16 hours |
| Expected Peg-out speed | ? | 1 hour | 33.3 hours | 0.16 hours |
| Custodian rotation | no | yes | no | yes |
| Fee structure | % of BTC moved | % of BTC moved | ? | Transaction fees |

Reply:
Thanks. Updated in 5e3d0de8dcb1530ac7e047115e43485954c401d3